ingest/src/main/java/com/connexta/ingest/client/StoreClient.java

```diff
@@ -71,8 +70,8 @@ public String getFilename() {
     final URI location;
     try {
       location = restTemplate.postForLocation(storeEndpoint, request);
-    } catch (final RestClientException e) {
-      throw new StoreException("Unable to POST to store endpoint", e);
+    } catch (Exception e) {
```

Comment:
Why do we catch all exceptions here? Is it possible that we could get an exception not caused by the Store Service being unavailable? Same question for the TransformClient.